### PR TITLE
Feature/61 ESLint Implementation For Github

### LINKED ISSUE #61: Implement lint on github

### PR DESCRIPTION
**Pull request by Manraj45**
resolves #61

### CODE REVIEW

File: bike-erp/backend/src/main/services/authenticationService/AuthenticationService.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class AuthenticationService {
         const account = await AuthenticationService.accountDao.fetchAccount(email);
 
         //Verifying if there was any account with the same email in the database
-        if (account.length == 0) {
+        if (account.length === 0) {
```
Comment (nasaku898) [inline bike-erp/backend/src/main/services/authenticationService/AuthenticationService.ts:44]: Good change on this point, should make the code more strict

**Review by nasaku898 — APPROVED** [review]
ESLint seems to be well implement and integrated into our github workflow. This is a great addition to make our code more solid and consistent. 
Great work! 💯

File: bike-erp/backend/src/main/services/authenticationService/AuthenticationService.ts (hunk 1 above)
Comment (jmezaber) [reply bike-erp/backend/src/main/services/authenticationService/AuthenticationService.ts:44]: I agree with Simon!

**Review by jmezaber — APPROVED** [review]
Great work as always Manraj!

**Pull request merged**